src/GitHub.js

```diff
@@ -6,8 +6,11 @@ export default class GitHub {
   octokit
   lastResponse = null
 
-  constructor(access_token) {
-    this.octokit = new Octokit({ auth: access_token })
+  constructor(access_token, host = "https://github.com/api/v3") {
+    this.octokit = new Octokit({ 
+      baseUrl: host,
```

Comment:
Added checks for non-empty and non-blank string. Otherwise it will assign undefined (default Octokit behaviour).

I know you already have Utils in "common/common.js" but I don't know how to export it with your current setup.